ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -110,7 +110,10 @@ <h3 class="title">
     *ngIf="currentObject!.type === 'tableName'"
     (selectedTabChange)="tabChanged($event)"
   >
-    <mat-tab label="COLUMNS">
```

Comment:
Table css needs to be fixed as borders are not of correct size: https://drive.google.com/file/d/1gDTaPfi5HWXuYtPzyfWbxM0EDpI5488D/view?usp=share_link&resourcekey=0-sQR3dRHA2FzfsDfmRgtt8Q

Reply:
Will be fixed in Bughunt-v7 PR

Reply:
will this css issue be handled in v7 PR- https://drive.google.com/file/d/1nfR-nYMy1UZCok3E956qsIGjVI60SfUF/view?usp=share_link&resourcekey=0-ZcMEEc81cn8us4Jbs1fHaw ?